pre-docs/bolt_options.md

```diff
@@ -50,19 +50,19 @@ To pass nodes to Bolt in a file, pass the file name and relative location with
 the `--nodes` flag and an `@` symbol: `bolt command run --nodes @nodes.txt`
 
 To pass nodes on `stdin`, on the command line, use a command to generate a node
-list, and pipe the result to Bolt : `<COMMAND> | bolt command run --nodes` For
+list, and pipe the result to Bolt with `-` after `--nodes` : `<COMMAND> | bolt command run --nodes -` For
 example, if you have a node list in a text file, you might run `cat nodes.txt |
 bolt command run --nodes`
 
 
-### Specifying nodes in an inventory file
+### Specifying nodes from an inventory file
 
-To specify nodes in an inventory file, reference nodes by node name or the name
-a group of nodes.
+To specify nodes from an inventory file, reference nodes by node name, a glob
+matching names in the file or the name a group of nodes.
```

Comment:
`, or the name of a group of nodes`. Missing `of` and Oxford comma.

Reply:
Still missing `of`, as in `name of a group of nodes` instead of `name a group of nodes`.